packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -0,0 +1,427 @@
+      classNames.cell,
+      getFocusStyle(theme),
+      FontClassNames.small,
+      {
+        color: colors.headerForegroundColor,
+        position: 'relative',
+        display: 'inline-block;',
+        boxSizing: 'border-box',
+        padding: `0 ${values.cellPadding}`,
+        border: 'none',
+        lineHeight: 'inherit',
+        margin: '0',
+        height: values.rowHeight,
+        verticalAlign: 'top',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        textAlign: 'left',
+        selectors: {
+          '&:hover $gripperBarVerticalStyle': {
+            display: 'block'
+          },
+          '& $filterChevron': {
+            color: colors.dropdownChevronForegroundColor,
+            paddingLeft: 4,
+            verticalAlign: 'middle'
+          },
+          $collapseButton: {
+            paddingRight: 0
+          }
+        }
+      }
+    ],
+
+    gripperBarVerticalStyle: [
+      {
+        display: 'none',
+        position: 'absolute',
+        textAlign: 'left',
+        color: palette.neutralTertiary,
+        left: 1
+      }
+    ],
+
+    cellSizer: [
+      classNames.cellSizer,
+      focusClear(),
+      {
+        display: 'inline-block',
+        position: 'relative',
+        cursor: 'ew-resize',
+        bottom: 0,
+        top: 0,
+        overflow: 'hidden',
+        height: 'inherit',
+        background: 'transparent',
+        zIndex: 1,
+        width: 16,
+
+        selectors: {
+          '&:after': {
+            content: '',
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            width: 1,
+            background: colors.resizerColor,
+            opacity: 0,
+            left: '50%'
+          },
+
+          '&:focus:after': cellSizerFadeInStyles,
+          '&:hover:after': cellSizerFadeInStyles,
+          '&$cellIsResizing:after': {
+            ...cellSizerFadeInStyles,
+            ...{
+              boxShadow: '0 0 5px 0 0 rgba(0, 0, 0, 0.4)'
+            }
+          }
+        }
+      }
+    ],
+
+    cellIsResizing: [classNames.isResizing],
+
+    cellSizerStart: [
+      {
+        margin: '0 -8px'
+      }
+    ],
+
+    cellSizerEnd: [
+      {
+        margin: 0,
+        marginLeft: -16
+      }
+    ],
+
+    collapseButton: [
+      classNames.collapseButton,
+      {
+        textAlign: 'center',
+        transform: 'rotate(-180deg)',
+        transformOrigin: '50% 50%',
+        transition: 'transform 0.1s linear',
+        width: 20,
+        outline: 0
+      },
+      isAllCollapsed && {
+        transform: 'rotate(0deg)'
+      },
+      isAllCollapsed && classNames.isCollapsed
+    ],
+
+    iconOnlyHeader: [
+      {
+        selectors: {
+          $nearIcon: {
+            paddingLeft: 0
+          }
+        }
+      }
+    ],
+
+    nearIcon: [
+      {
+        color: colors.iconForegroundColor,
+        opacity: 1,
+        paddingLeft: 8,
+        selectors: {
+          '&$sortIcon': {
```

Comment:
I noticed some places are like this while others on line 283 are just the name without quotes. Is there a reason for the difference?

Reply:
$foo$bar is legal identifier in JS whereas &$sortIcon isn't but it is a legal key for an object.